src/optimizer/OptimizerUtils.cpp

```diff
@@ -274,7 +274,15 @@ Value OptimizerUtils::boundValueWithMax(const meta::cpp2::ColumnDef& col, const
             d.day = 31;
             return Value(d);
         }
-        case Value::Type::DATETIME : {
+        case Value::Type::TIME: {
+            Time dt;
+            dt.hour = 24;
```

Comment:
23

Reply:
> 23, the DateTime ditto

Thanks for your remind. I think the 24, 60 , 60 is correct. if we want range scan, the max value should be max + 1. for example :
rang(1, 3) , the result is 1, 2. so if we want result is 1,2,3 , the range pair must be {1, 4} .